wire-ios-sync-engine/Tests/Source/UnauthenticatedSession/URLActionProcessors/CompanyLoginURLActionProcessorTests.swift

```diff
@@ -28,7 +28,7 @@ final class CompanyLoginURLActionProcessorTests: ZMTBaseTest, UnauthenticatedSes
 
     override func setUp() {
         super.setUp()
-
+        DeveloperFlag.useWireAuthentication.enable(false)
```

Comment:
suggestion: I didn't see that in original PR, I guess some tests are failing here, but should be fixed with SSO.
```suggestion
        DeveloperFlag.useWireAuthentication.enable(false, storage: .temporary())
```